extensions/schema-compare/src/test/schemaCompare.test.ts

```diff
@@ -79,22 +84,22 @@ describe('SchemaCompareResult.start', function (): void {
 	it('Should start with the source as database', async function (): Promise<void> {
 		let sc = new SchemaCompareTestService();
 
-		let result = new SchemaCompareMainWindow(sc, mockExtensionContext.object);
-		await result.start({connectionProfile: mockConnectionProfile});
+		let result = new SchemaCompareMainWindow(testContext.apiWrapper.object, sc, mockExtensionContext.object);
+		await result.start({connectionProfile: mockIConnectionProfile});
 		let promise = new Promise(resolve => setTimeout(resolve, 5000)); // to ensure comparison result view is initialized
 		await promise;
 
 		should.notEqual(result.sourceEndpointInfo, undefined);
 		should.equal(result.sourceEndpointInfo.endpointType, mssql.SchemaCompareEndpointType.Database);
-		should.equal(result.sourceEndpointInfo.serverName, mockConnectionProfile.serverName);
-		should.equal(result.sourceEndpointInfo.databaseName, mockConnectionProfile.databaseName);
+		should.equal(result.sourceEndpointInfo.serverName, mockIConnectionProfile.serverName);
+		should.equal(result.sourceEndpointInfo.databaseName, mockIConnectionProfile.databaseName);
 		should.equal(result.targetEndpointInfo, undefined);
 	});
 
 	it('Should start with the source as dacpac.', async function (): Promise<void> {
 		let sc = new SchemaCompareTestService();
 
-		let result = new SchemaCompareMainWindow(sc, mockExtensionContext.object);
+		let result = new SchemaCompareMainWindow(testContext.apiWrapper.object, sc, mockExtensionContext.object);
 		const dacpacPath = 'C:\\users\\test\\test.dacpac';
```

Comment:
seems weird to have the windows path (if this is expected can you add a comment to explain this please) - I know not a part of your PR

Reply:
It is just a dummy file path. Updated it to 'test.dacpac'- so won't be platform specific.